### ISSUE TITLE
Gerber.zip

### ISSUE THREAD

**Issue by MY20-PHEV** [stuartpittaway/WebastoWBUSInterface#4]
Hi Stuart,
First off, thanks for everything you have done on this project. Im looking to get the board done by JLCPCB and wanted them to do the assembly if possible but Im a bit of a noob and I can't see a list of components for the SMD assembly service inside the Gerber.zip ? Am I missing something or is this not available ? Thanks in advance.

**Comment by stuartpittaway** [comment]
I hand soldered these boards, many years ago!  I didn't really touch the project since (although its still sitting on a shelf!)

**Comment by janrunejohansen** [comment]
The footprints for the NPN transistors are wrong. Base and Emitter are the wrong way around. So the PCB doesnt work.

**Comment by stuartpittaway** [comment]
Hi @janrunejohansen, as you can see from the repository, I've not touched this project for over 8 years now.  I did get some boards made, but never had chance to use them.

I'd be delighted if you care to fix the issues you have found and submit a pull request to GITHUB.
Thanks.

**Comment by janrunejohansen** [comment]
Yeah i get that. I was just letting people know just in case anyone came to order a new batch 🙂 I have fixed mine with inverted transistors, so got what i need for now. Im not into Kicad so will be a bit work for me...
